graphql/src/main/java/io/micronaut/configuration/graphql/ws/apollo/GraphQLApolloWsSender.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micronaut.configuration.graphql.apollo.ws;
+package io.micronaut.configuration.graphql.ws.apollo;
```

Comment:
don't change the package unless we need to that could be an unnecessary breaking change